File: tools/bench/answers/go_sonnet/s2/06_stats.go
package main

import "fmt"

func main() {
	nums := []int{4, 8, 15, 16, 23}
	min, max, sum := nums[0], nums[0], 0
	for _, n := range nums {
		if n < min {
			min = n
		}
		if n > max {
			max = n
		}
		sum += n
	}
	fmt.Println(min)
	fmt.Println(max)
	fmt.Println(float64(sum) / float64(len(nums)))
}

File: tools/bench/answers/rust_sonnet/s2/06_stats.rs
fn main() {
    let list = [4, 8, 15, 16, 23];
    let min = list.iter().copied().min().unwrap();
    let max = list.iter().copied().max().unwrap();
    let sum: i64 = list.iter().sum();
    let mean = sum as f64 / list.len() as f64;
    println!("{}", min);
    println!("{}", max);
    println!("{}", mean);
}
